src/components/SeriesInCatalog.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Card, Button } from "react-bootstrap";
+import "@/styles/CatalogPage.scss";
+
+interface SeriesProps {
+  name: string;
+  url: string;
+}
+
+const SeriesInCatalog = ({ name, url }: SeriesProps) => {
```

Comment:
```suggestion
const CatalogItem = ({ name, url }: SeriesProps) => {
```

Reply:
And probably, it's better for extensibility to use the object as a prop instead of the amount of props with the same context.